server/chat-plugins/modlog.ts

```diff
@@ -44,6 +44,7 @@ const LONG_QUERY_DURATION = 2000;
 const LINES_SEPARATOR = 'lines=';
 const MAX_RESULTS_LENGTH = MORE_BUTTON_INCREMENTS[MORE_BUTTON_INCREMENTS.length - 1];
 const LOG_PATH = 'logs/modlog/';
+const IPS_REGEX = /[([]([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)[)\]]/g;
```

Comment:
`\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}` seems more desirable, or at the very least `\d+` instead of `[0-9]+` (though obviously you didnt write this regex)

Reply:
from chat, maybe `[0-9]{1,3}` is more desirable